### PR TITLE
Remove getResourceOperation for performGet.

### PR DESCRIPTION
**Pull request by bryanhelmig**
Is definitely a breaking change - will need to update other docs and example apps.

### CODE REVIEW

File: docs/build/schema.md [hunk 1]
```diff
@@ -195,6 +196,31 @@ Key | Required | Type | Description
...
+
+
+
+#### Properties
+
+Key | Required | Type | Description
+--- | -------- | ---- | -----------
+`resource` | no | [/RefResourceSchema](#refresourceschema) | Optionally reference and extends a resource. Allows us to automatically tie together samples, lists and hooks greatly improving the UX. EG: if you had another trigger reusing a resource but filtering the results.
+`perform` | **yes** | oneOf([/RequestSchema](#requestschema), [/FunctionSchema](#functionschema)) | How will we get the data? This can be a function like `(z) => [{id: 123}]` or a request like `{url: 'http...'}`.
+`performGet` | no | oneOf([/RequestSchema](#requestschema), [/FunctionSchema](#functionschema)) | How will we get a single bit of rich data? If you find yourself reaching for this - consider resources and their built-in get methods.
```
Comment (BrunoBernardino) [inline docs/build/schema.md:217]: Maybe add a note this is necessary for public apps with REST hooks?

File: docs/build/schema.md [hunk 2]
```diff
@@ -195,6 +196,31 @@ Key | Required | Type | Description
...
+`inputFields` | no | [/DynamicFieldsSchema](#dynamicfieldsschema) | What should the form a user sees and configures look like?
+`outputFields` | no | [/DynamicFieldsSchema](#dynamicfieldsschema) | What fields of data will this return? Will use resource outputFields if missing, will also use sample if available.
+`sample` | no | `object` | What does a sample of data look like? Will use resource sample if missing.
```
Comment (BrunoBernardino) [inline docs/build/schema.md:220]: Potentially change `Will use resource sample` to `Will use resource or performGet sample`?

File: lib/schemas/BasicActionOperationSchema.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+// or maybe use allOf which is built into json-schema
+const BasicActionOperationSchema = JSON.parse(JSON.stringify(BasicOperationSchema.schema));
+
+BasicActionOperationSchema.id = '/BasicActionOperationSchema';
+
+BasicActionOperationSchema.properties = {
+  resource: BasicActionOperationSchema.properties.resource,
+  perform: BasicActionOperationSchema.properties.perform,
+  performGet: {
+    description: 'How will we get a single bit of rich data? If you find yourself reaching for this - consider resources and their built-in get methods.',
```
Comment (BrunoBernardino) [inline lib/schemas/BasicActionOperationSchema.js:19]: If the comment above is updated, update this one as well, please.

**Review by BrunoBernardino — APPROVED** [review]
Looks good, I just have some comments around help texts.

File: docs/build/schema.md (hunk 1 above)
Comment (bcooksey) [reply docs/build/schema.md:217]: `performList` will be, but not this one, correct?

Comment (BrunoBernardino) [reply docs/build/schema.md:217]: That's right.

File: docs/build/schema.md (hunk 2 above)
Comment (bcooksey) [reply docs/build/schema.md:220]: This sample is the only one for the operation, and we won't try to do a performGet to fetch more data, so I think this is correct.

**Pull request merged**